general_rules/Organization.tex

```diff
@@ -29,6 +29,11 @@ \subsection{Stage system}\label{rule:stages}
 
 In case of having no considerable score deviation between a team advancing to the next stage and a team dropping out, the TC may announce additional teams advancing to the next stage.
 
+\subsection{Stage II for SSPL and DSPL}
+
+SSPL and DSPL teams may attempt Stage I tests in Stage II, in addition to any of the Stage II tests.
+
+As Stage I tasks have a maximum reward of 1000 points, if those tasks are used in Stage II, their rewards (and penalties) are doubled.
```

Comment:
You should only be able to get half of the possible points if you do a stage 1 task in stage 2. I would personally add OPL in there just to be fair for every league.

Reply:
This math just doesn't work. If Stage 1 tasks are used in Stage 2, they should be given 0 points because they're not solving Stage 2 tasks, or they should get some minuscule fraction of points, and some multiplicative factor on the penalties.

If not ZERO, then I would suggest:

Stage I can be used in Stage II for 1/10th of their original point value, and 10x the penalty value...

Basically: you better do it perfectly or just go and solve a Stage II task.